src/groups/bmq/bmqt/bmqt_subscription.cpp

```diff
@@ -43,7 +43,7 @@ const int Subscription::k_DEFAULT_CONSUMER_PRIORITY        = 0;
 
 unsigned int SubscriptionHandle::nextId()
 {
-    static bsls::AtomicUint s_id = 0;
+    static bsls::AtomicUint s_id = k_INVALID_HANDLE_ID;
```

Comment:
https://github.com/bloomberg/blazingmq/issues/141

I created an issue about this line. It's out of the scope of this PR and could be fixed in another PR.